cocotb/config.py

```diff
@@ -61,26 +60,50 @@ def __call__(self, parser, namespace, values, option_string=None):
         parser.exit()
 
 
-def get_parser():
```

Comment:
Please add back `get_parser`, it's needed to make the docs work.

Reply:
:+1: